scipy/io/arff/arffread.py

```diff
@@ -177,6 +173,8 @@ def get_date_format(atrv):
     if m:
         pattern = m.group(1).strip()
         # convert time pattern from Java's SimpleDateFormat to C's format
+        # FIXME the Java spec appears to allow different numbers of repeating
+        # letters than just the ones hardcoded below
```

Comment:
In the interest of keeping this PR small / low-risk / easy to review, I've just noted this rather than try to fix it.